vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps

```diff
@@ -98,3 +98,81 @@ When I enter `<text>` in field located `By.xpath(<xpath>)`
 
 Composite: When I select '$text' from a drop down by the xpath '$xpath'
 When I select `$text` from drop down located `By.xpath(<xpath>)`
+
+Composite: Then an element with the name '$elementName' containing text '$text' exists
+Then number of elements found by `By.elementName(<elementName>):a->filter.TextPart(<text>)` is equal to `1`
+
+Composite: Then an element with the name '$elementName' and text '$text' exists
+Then number of elements found by `By.elementName(<elementName>)->filter.caseSensitiveText(<text>)` is equal to `1`
+
+Composite: Then a [$state] element with the name '$elementName' and text '$text' exists
+Then number of elements found by `By.elementName(<elementName>)->filter.caseSensitiveText(<text>).state(<state>)` is equal to `1`
+
+Composite: Then an element with the name '$elementName' exists
+Then number of elements found by `By.elementName(<elementName>)` is equal to `1`
+
+Composite: Then an element with the name '$elementName' does not exist
+Then number of elements found by `By.elementName(<elementName>)` is equal to `0`
+
+Composite: Then an element with the xpath '$xpath' does not exist
+Then number of elements found by `By.xpath(<xpath>)` is equal to `0`
+
+Composite: Then a [$state] element with the name '$elementName' exists
+Then number of elements found by `By.elementName(<elementName>):a->filter.state(<state>)` is equal to `1`
+
+Composite: Then an element with the tag '$elementTag' and text '$text' exists
```

Comment:
Why 'the tag', but just 'text' ?

Reply:
because this composite is just converted Java step, its syntax must be preserved to do not introduce breaking changes